packages/chart/src/SummaryC.ts

```diff
@@ -0,0 +1,203 @@
+
+    constructor() {
+        super();
+        this.playInterval(this.playInterval());
+    }
+
+    currentRow() {
+        return this.data()[this._playIntervalIdx];
+    }
+
+    enter(_domNode, element) {
```

Comment:
Should be "domNode" not _domNode - the underbar was harks back to when there was a lint test for unused params...